### DISCUSSION TITLE
Creating a progress bar beneath the URLbar

### DISCUSSION THREAD

**Discussion by clifford269** [xiaoxiaoflood/firefox-scripts#321] [Legacy Development]
Is it possible? If so, please help me with it.

**Comment by 117649** [comment]
https://addons.mozilla.org/en-US/firefox/addon/load-progress-bar/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search

**Reply by clifford269** [reply]
Beneath the urlbar...
